cmd/containerd/main.go

```diff
@@ -120,11 +120,13 @@ high performance container runtime
 		)
 		switch runtime {
 		case "runc":
+			logrus.Info("containerd: run with runc executor")
```

Comment:
I think we should not change the format of logrus output, because Docker will redirect containerd logs to docker logs, WDYT?